Embd_Chatbot_adithya/chatbot/.env.example

```diff
@@ -0,0 +1,33 @@
+# Medical AI Chatbot
```

Comment:
This folder structure is untraditional

Reply:
Embd_Chatbot_adithya is not a good name.  I am not sure why this is here either when there is a README.md